contracts/onft/LzAppLite.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import { ILayerZeroReceiver } from "@layerzerolabs/solidity-examples/contracts/interfaces/ILayerZeroReceiver.sol";
+import { ILayerZeroUserApplicationConfig } from "@layerzerolabs/solidity-examples/contracts/interfaces/ILayerZeroUserApplicationConfig.sol";
+import { ILayerZeroEndpoint } from "@layerzerolabs/solidity-examples/contracts/interfaces/ILayerZeroEndpoint.sol";
+import { BytesLib } from "@layerzerolabs/solidity-examples/contracts/util/BytesLib.sol";
+
+/*
+ * a generic LzReceiver implementation
+ */
+abstract contract LzAppLite is Ownable, ILayerZeroReceiver, ILayerZeroUserApplicationConfig {
```

Comment:
Would it make sense to list out what makes this `Lite` vs. the non-Lite implementation?

Reply:
Good call. Added.

Reply:
Would it make sense to also add these kinds of comments to the other 'Lite' contracts?